### PR TITLE
2060 acceptance review

### PR DESCRIPTION
**Pull request by ijiraq**
I've reviewed the code for purpose of acceptance and made a few changes to help ensure dev install works smoothly and to get rid of some rough edges.   Please have a look and see if we have a net improvement here.

### CODE REVIEW

File: vos/vos/tests/test_vofile.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from builtins import object
+from future.builtins import object
```
Comment (andamian) [inline vos/vos/tests/test_vofile.py]: astropy recommends the six library as a way to link Python 2 and 3 and this is what I was trying to always use. future, however, seems to be more appropriate if 2.7 is the earliest supported version. Should we start using future then? I would say either one or the other. Do you have more experience with future?

**Comment by ijiraq** [comment]
six is fine with me.

but the previous was 

except:
  builtins = __builtins__ 

from setup.py which didn't rely on either.


JJ

**Comment by andamian** [comment]
Using README.rst is a good idea. I don't understand why, on pypi, is not formatted properly (see https://testpypi.python.org/pypi/vos/). I'll play with it tomorrow. Time to get introduced to Sphinx.

**Comment by andamian** [comment]
And we should do the same with vofs.

**Pull request merged**